src/openfermion/utils/_trotter_qasm_conversion_test.py

```diff
@@ -0,0 +1,183 @@
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+
+
+'''
+Python's 'unittest' is run with either:
+* python {this file's name}.py
+* python -m unittest discover simple_example  # (for "discovering"
+		all test files)
+'''
+
+
+
+
+import unittest
+import os
+import cStringIO
+
+from openfermion.ops import QubitOperator
+#import openfermion.utils._trotter_qasm_conversion as tqc
+from openfermion.utils._trotter_qasm_conversion import *
+#from _trotter_qasm_conversion import *
+
+
+class TrottQasmTest(unittest.TestCase):
+
+	def setUp(self):
+		# First qubit operator example
+		self.qo1 = QubitOperator('X0 Z1 Y3', 0.5) + 0.6 * QubitOperator('Z3 Z4')
+
+
+	def test_simple1_trot1(self):
+
+
+		# strioQasm = cStringIO.StringIO()
+		strioQasm = "test.txt"
```

Comment:
I recommend choosing a file extension such as '.qasm', that can be used as a standard throughout qasm use in openfermion or perhaps 'rqasm' for restricted qasm.